README.md

```diff
@@ -1,2 +1,77 @@
 # hugo-course-publisher
-hugo research
+
+This implements OCW course sites using the [Hugo](https://gohugo.io/) static
+site generator.  The content is provided by
+[ocw-to-hugo](https://github.com/mitodl/ocw-to-hugo), and CSS and JS is built
+for the site using webpack.
+
+## installation
+
+To install and run the site locally you can either use docker or run it
+natively.
+
+### docker
+
+To run the site in docker you'll need docker and docker-compose. With those in
+place you should be able to simply run:
+
+```sh
+docker-compose up
+```
+
+and the site should be available at <http://localhost:3000>. Note that after the
+docker container comes up it may take a few minutes for the hugo site to finish
+rendering. This is because we are rendering a pretty large number of Markdown
+files, and a few UI elements are particularly heavy to render.
+
+### without docker
+
+If you prefer not to use docker you can also simply do:
+
+```sh
+yarn install --pure-lockfile
+npm start
```

Comment:
I believe you'll need to run `npm build` at this point before `npm start`?  Also, on macOS at least, I'm getting the following error running `npm start` or `npm run start:hugo`:

```
c4103-pc-mac:hugo-course-publisher c4103$ npm run start:hugo

> hugo-course-publisher@1.0.0 start:hugo /Users/c4103/Github/hugo-course-publisher
> hugo server -s site -p 3000 --bind 0.0.0.0

events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: spawn /Users/c4103/Github/hugo-course-publisher/node_modules/hugo-bin/vendor/hugo ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:270:12)
    at onErrorNT (internal/child_process.js:456:16)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn /Users/c4103/Github/hugo-course-publisher/node_modules/hugo-bin/vendor/hugo',
  path: '/Users/c4103/Github/hugo-course-publisher/node_modules/hugo-bin/vendor/hugo',
  spawnargs: [
    'server',  '-s',
    'site',    '-p',
    '3000',    '--bind',
    '0.0.0.0'
  ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hugo-course-publisher@1.0.0 start:hugo: `hugo server -s site -p 3000 --bind 0.0.0.0`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the hugo-course-publisher@1.0.0 start:hugo script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/c4103/.npm/_logs/2020-07-01T16_37_58_732Z-debug.log
```

The site successfully starts if I manually run `hugo server -s site -p 3000 --bind 0.0.0.0`,  but for some reason I get that error running it directly through the npm command.

Reply:
hmm I don't see that issue so it's going to be tricky to debug since I don't have a mac - any idea what the problem might be? also did you `rm -rf node_modules` first? because it could be that you have a linux binary installed or something

Reply:
That is a good point, as was actually just thinking that before I read this.  I'll try it again and let you know.  Either way, I think the `npm build` will be necessary to make sure pdf.js gets built.

Reply:
Confirmed, I just had to blow away `node_modules`.  I think the `npm build` step is still necessary though?

Reply:
yeah, I added it in